tests/conftest.py

```diff
@@ -62,6 +63,26 @@ def increment_cache_key_prefix():
     cache.key_prefix = "t{0}".format(key_prefix_counter)
 
 
+@pytest.fixture(scope="session", autouse=True)
+def block_unmocked_requests():
```

Comment:
You should be able to just add ``monkeypatch`` as a param here and not have to instantiate or import it like this test:  https://github.com/mozilla/treeherder/blob/master/tests/conftest.py#L260-L260